doc/docs.md

```diff
@@ -1993,20 +1993,17 @@ mut p := Point{
 	x: 10
 	y: 20
 }
-p = Point{
-	x: 30
-	y: 4
-}
+
+// The field name can be omitted.
+p = Point{30, 4}
+
 assert p.y == 4
-//
+
 // array: first element defines type of array
-points := [Point{10, 20}, Point{20, 30}, Point{40, 50}]
-println(points) // [Point{x: 10, y: 20}, Point{x: 20, y: 30}, Point{x: 40,y: 50}]
+// points := [Point{10, 20}, Point{20 30}] // You can omit the `,` between the parameters, as in the 2nd case.
+// println(points) // [Point{x: 10, y: 20}, Point{x: 20, y: 30}]
```

Comment:
Why do you comment the lines?
Doing so in effect disables the automatic checking that `v check-md .` does.

Reply:
I think the compiler is right and it's a bug in the fmt.

Reply:
If `p = Point{30, 4}` is added, these two lines is necessary?
And is  this comment `array: first element defines type of array` necessary for _this_ section?

Reply:
yes, they are necessary